compiler/sem/pragmas.nim

```diff
@@ -1324,8 +1318,6 @@ proc applySymbolPragma(c: PContext, sym: PSym, it: PNode): PNode =
       of wNoInit:
         result = noVal(c, it)
         incl(sym.flags, sfNoInit)
-      of wCodegenDecl:
```

Comment:
I think this needs to stay, but with a `c.config.backend` check against `backendJs` or else error?

Reply:
Yep, it does. The original changeset fully removed the pragma, and this is one place where I missed to revert the change.